### PR TITLE
Mobile 4017

### PR DESCRIPTION
**Pull request by alfonso-salces**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
         "moodle.config.json": "jsonc",
         "moodle.config.*.json": "jsonc",
     },
+    "files.eol": "\n",
```
Comment (NoelDeMartin) [inline .vscode/settings.json:25]: I think this should go in the section above (called "Formatting").

**Review by NoelDeMartin — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (alfonso-salces) [reply .vscode/settings.json]: Done! thank you

File: src/core/classes/tests/error.test.ts [hunk 2]
```diff
@@ -30,15 +32,14 @@ describe('CoreError', () => {
         } catch (e) {
             error = e;
         }
-
```
Comment (NoelDeMartin) [inline src/core/classes/tests/error.test.ts:33]: I believe you've removed this line by mistake?

File: src/testing/utils.ts [hunk 3]
```diff
@@ -245,3 +246,12 @@ export async function renderWrapperComponent<T>(
 
     return renderTemplate(component, `<${tag} ${inputAttributes}></${tag}>`, config);
 }
+/**
```
Comment (NoelDeMartin) [inline src/testing/utils.ts:249]: I think the JSDocs should be added in the same commit as the method, it'd also be better to have an empty line between functions (I've added a new linting rule to enforce that in https://github.com/moodlehq/moodleapp/pull/3214, but it's not merged yet)

**Review by NoelDeMartin — CHANGES_REQUESTED** [review]

File: src/core/classes/tests/error.test.ts (hunk 2 above)
Comment (alfonso-salces) [reply src/core/classes/tests/error.test.ts]: Nice catch! thanks you

File: src/testing/utils.ts (hunk 3 above)
Comment (alfonso-salces) [reply src/testing/utils.ts:250]: Done!

**Review by NoelDeMartin — APPROVED** [review]
I see you ended up squashing both commits, I think it was fine with two but I'll go ahead with merging anyways so that we get this done :).

**Pull request merged**